### PR TITLE
vector layer is made but not added to project

### PR DESCRIPTION
**Pull request by thayeray**
Changed the text to add the vector layer to the project after creating it.

Goal:

Ticket(s): #

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: docs/pyqgis_developer_cookbook/loadlayer.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ them here.
 Vector Layers
 =============
 
-To create a vector layer instance, specify layer's data source identifier, name for the
+To create and add a vector layer instance to the project, specify layer's data source identifier, name for the
```
Comment (havatv) [inline docs/pyqgis_developer_cookbook/loadlayer.rst:49]:
```suggestion
To create and add a vector layer instance to the project, specify the layer's data source
identifier, name for the layer and provider's name:
```

File: docs/pyqgis_developer_cookbook/loadlayer.rst [hunk 2]
```diff
@@ -46,7 +46,7 @@ them here.
...
 layer and provider's name:
```
Comment (havatv) [inline docs/pyqgis_developer_cookbook/loadlayer.rst:50]:
```suggestion
```

File: docs/pyqgis_developer_cookbook/loadlayer.rst [hunk 3]
```diff
@@ -245,6 +255,7 @@ providers:
       # Use project's transform context
       provider_options.transformContext = QgsProject.instance().transformContext()
       vlayer.setDataSource(uri, "layer name you like", "WFS", provider_options)
+      QgsProject.instance().addMapLayer(vlayer)
```
Comment (havatv) [inline docs/pyqgis_developer_cookbook/loadlayer.rst:258]: Given that this existing layer is already added, is the `addMapLayer` necessary?

**Comment by thayeray** [comment]
Good catch, thanks for proof reading!  

    On Tuesday, April 28, 2020, 03:25:28 AM EDT, Håvard Tveite <notifications@github.com> wrote:  
 
 


@havatv approved this pull request.

Thanks. This looks fine to me. I have some suggestions / questions.

In docs/pyqgis_developer_cookbook/loadlayer.rst:
> @@ -46,7 +46,7 @@ them here.
 Vector Layers
 =============
 
-To create a vector layer instance, specify layer's data source identifier, name for the
+To create and add a vector layer instance to the project, specify layer's data source identifier, name for the
⬇️ Suggested change-To create and add a vector layer instance to the project, specify layer's data source identifier, name for the
+To create and add a vector layer instance to the project, specify the layer's data source
+identifier, name for the layer and provider's name:

In docs/pyqgis_developer_cookbook/loadlayer.rst:
> @@ -46,7 +46,7 @@ them here.
 Vector Layers
 =============
 
-To create a vector layer instance, specify layer's data source identifier, name for the
+To create and add a vector layer instance to the project, specify layer's data source identifier, name for the
 layer and provider's name:
⬇️ Suggested change-layer and provider's name:

In docs/pyqgis_developer_cookbook/loadlayer.rst:
> @@ -245,6 +255,7 @@ providers:
       # Use project's transform context
       provider_options.transformContext = QgsProject.instance().transformContext()
       vlayer.setDataSource(uri, "layer name you like", "WFS", provider_options)
+      QgsProject.instance().addMapLayer(vlayer)

Given that this existing layer is already added, is the addMapLayer necessary?

**Comment by havatv** [comment]
> Good catch, thanks for proof reading!

You are welcome, @thayeray. You can accept suggestions by going to the *Files changed* "tab" and either commit them one by one (*Commit suggestion*), or (better) accumulate them (*Add suggestion to batch*) and commit them in one go (*Commit suggestions*).

**Comment by havatv** [comment]
> Not sure how to make the 3rd change you suggest. You are correct that what I added was unnecessary for the project's transform context.

You can edit in the *Files changed* tab by pressing the blue button with the white plus sign that appears when you move the cursor over the line you would like to edit. Once you have edited, you just commit the change.
You can also edit the file by choosing *Edit file* in the three dot (...) pull-down menu top right.

**Comment by thayeray** [comment]
Thanks for your patience. I think I got it.

**Comment by havatv** [comment]
> Thanks for your patience. I think I got it. — You are receiving this because you were mentioned. 

You are welcome. We very much appreciate your contribution - and are hoping for more! :+1

**Pull request merged**